### PR TITLE
users: Replace `w` and utmp watching with logind API

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: pkg/users/users.js [hunk 1]
```diff
@@ -174,12 +174,17 @@ async function getLogins() {
     const lastlog = await cockpit.spawn([LastLogPath], { environ: ["LC_ALL=C"] });
 
     let currentLogins = [];
+    const logind = cockpit.dbus("org.freedesktop.login1");
```
Comment (jelly) [inline pkg/users/users.js:177]: Don't want to push for this to happen in this PR but, I'd love to also get watching happening via logind:

Replacing:
```
        let handleUtmp;
        if (utmppath !== null) {
            // Watch `/var/run/utmp` or `/var/lib/wtmpdb/wtmp.db` to register when user logs in or out
            handleUtmp = cockpit.file(utmppath, { superuser: "try", binary: true });
            handleUtmp.watch(() => debouncedGetLogins(), { read: false });
        }
```

Comment (jelly) [reply pkg/users/users.js:177]: So we have a users global logind handle with signals for that.

**Review by jelly — APPROVED** [review]

**Comment by jelly** [comment]
Seeing all the test failures, this could be due to what I saw with the new systemd utmp triggers => systemd still is still busy cleaning up the session. So it is probably better to watch signals but **note**

On the new systemd you have two types of session (same as versions before but now exposed in list-sessions)
* User Manager session for user 1000
* User 1000's session

So likely you will see two RemoveSession's or RemoveUser. (forgot the exact wording).

**Comment by martinpitt** [comment]
You are right, when this happens, `loginctl list-users` says "1000 admin no     closing". But that goes away after a few seconds, but then there's no further event which would update the page -- so even increasing the wait timeout doesn't help. So (1) this should probably check for `active` (but then that requires a lot more calls), and/or we indeed do the listening to signals thing -- we want to do that anyway, and it may suffice. I'll look into that.

**Comment by martinpitt** [comment]
Updated to check the `State` property, the test is fine now.

I'll look into replacing that wtmp watch now.

**Comment by martinpitt** [comment]
@Lunarequest FYI -- this essentially reverts your recent work, but I think this is more robust, more compatible, and easier.

**Comment by Lunarequest** [comment]
I'm all for this if it can make this more reliable

**Review by jelly — APPROVED** [review]

**Comment by jelly** [comment]
Thanks, this is a great cleanup!

**Pull request merged**